### DISCUSSION TITLE
Documentation Wiki

### DISCUSSION THREAD

**Discussion by clayton1964** [pH7Software/pH7-Social-Dating-CMS#1087] [Ideas/Suggestions]
Hello all,

I discovered this project and immediately wanted to help.

While I have been setting up my own running site, I've been going through the documentation available and have noticed some things. 

I was wondering what everyone's thoughts would be about the creation of a solid documentation site? I have experience using MediaWiki and think it would be great for the documentation of ph7. An example of a MediaWiki site I maintain is located at [http://www.nparchistory.ca/](http://www.nparchistory.ca/). I would be willing to setup the initial site to get it up and running.

I would also be willing to host the site on one of my servers at no cost to the project. I run a small hosting company and have ample space on my systems for the site.

Comments?

(Edit: I just realized that the current doc site is done using a script written by the same author of ph7. I do not mean to step on any toes here. I would also be more than willing to help in the development of Nav-Doc-Script-V2. Once again, sorry if I offended anyone.)

Clayton

**Comment by rthuey2** [comment]
interested in what your site looks like these days
